changelog.json

```diff
@@ -1,6 +1,7 @@
 {
   "releases": {
     "0.7.1": [],
+    "0.7.1-beta5": ["[Fixed] Revert Updater workaround - #2350"],
```

Comment:
@joshaber any feels on how we should manage the release notes for beta releases? Also, this task is purely tech debt so I'm not sure whether it should be listed here.